### PR TITLE
Front end sites v1 pipeline docs updates

### PR DESCRIPTION
**Pull request by stevector**
## Summary

This pull request documents changes to the build pipeline of Front-End Sites. Before merging, I would like to fully cut over the Docs site itself to the v1 pipeline. We've had a few branches and test PRs run on it seemingly successfully for a few weeks.

## Effect

* Adds a new page specific to the [v1 pipeline](https://pr-8737-documentation.appa.pantheon.site/guides/decoupled/overview/v1-pipeline)
* Modifies a few related pages

## Remaining Work and Prerequisites

- [x] Switch the docs site itself to use `ANY` as the value for this environment variable as a show of confidence in the change.

### Dependencies and Timing

The product work is already done. @jsulz and I want this PR merged by Friday, November 3rd at the latest because there will be an email sent to customers starting on Monday, November 6th, referencing this documentation. The sooner we can make functional links, the better.

### CODE REVIEW

**Comment by stevector** [comment]
@jsulz can you review?

File: source/images/decoupled/purge-build-cachge--trigger.png
Comment (jsulz) [inline source/images/decoupled/purge-build-cachge--trigger.png:1]: I would update this screenshot with something in production. I was grabbing this from a Figma file, which also included the CDN cache purging work, but this is not available to customers.

Comment (stevector) [reply source/images/decoupled/purge-build-cachge--trigger.png:1]: Will do. And I see I mispelled a word in this file name

File: source/content/guides/decoupled/overview/10-v1-pipeline.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+innav: [false]
+categories: [create]
+cms: [decoupled]
+audience: [development]
+product: [decoupled]
+integration: [--]
+---
+
+In the Fall of 2023 Pantheon introduced a new build pipeline for Front End Sites.
+Among under-the-hood-optimizations, like [caching of folders between builds](/guides/decoupled/overview/manage-settings#build-cache), [the new pipeline now uses your `package.json` file](/guides/decoupled/overview/manage-settings#nodejs-version) to select the appropriate version of Node.js your build and your runtime environments.
```
Comment (rachelwhitton) [inline source/content/guides/decoupled/overview/10-v1-pipeline.md:21]: > to select the appropriate version of Node.js your build and your runtime environments.

missing for, maybe? 

> to select the appropriate version of Node.js for your build and your runtime environments.

Comment (stevector) [reply source/content/guides/decoupled/overview/10-v1-pipeline.md:21]: good catch!

File: source/images/decoupled/purge-build-cachge--trigger.png
Comment (stevector) [reply source/images/decoupled/purge-build-cachge--trigger.png:1]: done!

**Comment by stevector** [comment]
@rachelwhitton can you review one more time and then merge tomorrow or Wednesday?

File: source/content/guides/decoupled/overview/05-manage-settings.md [hunk 2]
```diff
@@ -31,13 +31,36 @@ You can change the Site Name, connect or disconnect your Git repository, and del
 
 ### Node.js Version
 
-Pantheon respects the setting in `.nvmrc` when selecting the Node.js version for runtime. Currently supported versions are:
+As of the release of our [V1 build pipeline](/guides/decoupled/overview/v1-pipeline), Front-End Sites runs can run v16, v18, or v20 of Node.js as specified in your `package.json` file.
+
+Non-LTS versions like v17 are not available.
```
Comment (rachelwhitton) [inline source/content/guides/decoupled/overview/05-manage-settings.md:36]: > Non-LTS versions like v17 are not available.

I suggest removing this sentence or revising it for accuracy based on https://nodejs.org/en/about/previous-releases  (which suggests v16 is also Non-LTS ?) 

If revising, I suggest adding context for "Non-LTS" maybe in a [tooltip](https://docs.pantheon.io/style-guide#tooltips)

File: source/content/guides/decoupled/overview/05-manage-settings.md [hunk 3]
```diff
@@ -31,13 +31,36 @@ You can change the Site Name, connect or disconnect your Git repository, and del
...
+
+Pantheon inspects the engines field in `package.json` when selecting a specific Node.js version for the build (Gatsby.js and Next.js) and runtime (Next.js) of your Front-End Site. For example, to select Node.js 16 for a given deployment, you should do the following:
+
+```json
```
Comment (rachelwhitton) [inline source/content/guides/decoupled/overview/05-manage-settings.md:40]: Make it more clear what file this excerpt is from by adding a title: https://docs.pantheon.io/style-guide#file-excerpts 
```suggestion
```json:title=package.json
```

File: source/content/guides/decoupled/overview/05-manage-settings.md [hunk 4]
```diff
@@ -31,13 +31,36 @@ You can change the Site Name, connect or disconnect your Git repository, and del
...
+Note that if you select a range of major Node.js versions, Pantheon will select the most recent Node.js major version for that deployment. For example, the following configuration would result in Node.js 18.x being selected:
+
+```json
+"engines": {
+  "node": ">=16.0.0 <20.0.0"
+},
+```
+We recommend that when you need to change the version of Node.js for your Front-End Site, that you test the new version by pushing the updated `package.json` to a branch first and create a pull request. Pantheon will create an environment for your site specific to that pull request.
+
+Sites made prior to DATE set the Node.js version via  `.nvmrc` and can select via:
```
Comment (rachelwhitton) [inline source/content/guides/decoupled/overview/05-manage-settings.md:57]: @stevector is DATE a placeholder here maybe?

File: source/content/guides/decoupled/overview/05-manage-settings.md [hunk 5]
```diff
@@ -122,6 +145,43 @@ Adjustments to these settings might be required if you are not using a starter k
 
 ![Build and Output Settings](../../../../images/fes-build-output-settings.png)
 
+## Build Cache
+
+The first build on any environment (production or either type of Multidevs) runs without a cache. During the first build, Pantheon caches the following assets based on the framework you’ve selected:
```
Comment (rachelwhitton) [inline source/content/guides/decoupled/overview/05-manage-settings.md:150]:
```suggestion
The first build on any environment ([production or either type of Multidevs](/guides/decoupled/overview/considerations#types-of-environments)) runs without a cache. During the first build, Pantheon caches the following assets based on the framework you’ve selected:
```

**Review by rachelwhitton — CHANGES_REQUESTED** [review]
Some minor suggestions requested for consideration

File: source/content/guides/decoupled/overview/05-manage-settings.md (hunk 4 above)
Comment (rachelwhitton) [reply source/content/guides/decoupled/overview/05-manage-settings.md:57]:
```suggestion
Sites made prior to DATE set the Node.js version via  `.nvmrc` and can select via:
```

**Review by rachelwhitton — APPROVED** [review]

**Pull request merged**